plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/StmtConversionContext.kt

```diff
@@ -73,3 +72,21 @@ fun <RTC : ResultTrackingContext> StmtConversionContext<RTC>.embedSetter(symbol:
         null, is FirDefaultPropertySetter -> getField(symbol)?.let { BackingFieldSetter(it) }
         else -> CustomSetter(embedFunction(setter.symbol))
     }
+
+fun StmtConversionContext<ResultTrackingContext>.getFunctionCallSubstitutionItems(
+    args: List<FirExpression>,
+): List<SubstitutionItem> = args.map { exp ->
+    when (exp) {
+        is FirLambdaArgumentExpression -> {
+            val anonExpr = exp.expression
+            if (anonExpr is FirAnonymousFunctionExpression) {
+                val lambdaBody = anonExpr.anonymousFunction.body!!
+                val lambdaArs = anonExpr.anonymousFunction.valueParameters.map { it.name }
+                SubstitutionLambda(lambdaBody, lambdaArs)
+            } else {
+                TODO("are there any other cases?")
```

Comment:
Is a named function expression another case? As in `::function`.

Reply:
That could be; I'm not changing this code at this point, just moving it around.  This is a place that will require more of a redesign soon.